integration/single-node-slug-deploy/check.go

```diff
@@ -384,6 +394,12 @@ func verifyHealthChecks(config *preparer.PreparerConfig, services []string) erro
 		} else if (res == kp.WatchResult{}) {
 			return fmt.Errorf("No results for %s", sv)
 		} else if res.Status != string(health.Passing) {
+			cmd := exec.Command("curl", "localhost:8500/v1/kv/health/hello?recurse")
```

Comment:
no reason I guess. These were mostly for debugging travis. Is it worth changing them to http get and ReadFile? or should I just remove them?